extensions/load-balancer/pom.xml

```diff
@@ -36,6 +36,7 @@
     <modules>
         <module>haproxy-extension</module>
         <module>nginx-extension</module>
+        <module>gce-extension</module>
         <module>lvs-extension</module>
     </modules>
 </project>
```

Comment:
It would be better to add a README.md file. Please run FindBugs or Code Analyze tool in IntelliJ IDEA and fix issues.
Thanks